common/src/main/java/org/apache/uniffle/common/rpc/ServerInterface.java

```diff
@@ -23,6 +23,8 @@ public interface ServerInterface {
 
   void start() throws IOException;
 
+  void startOnPort(int port) throws IOException;
```

Comment:
Could StreamServer extend ServerInterface, too?

Reply:
> Could StreamServer extend ServerInterface, too?

I think each server could extends ServerInterface, such as jetty server, so the ServerInterface could not in rpc package.